ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/tests/ActiveContractsService.scala

```diff
@@ -24,12 +24,14 @@ import com.daml.ledger.test_stable.Test.{
 import io.grpc.Status
 import scalaz.syntax.tag._
 
+import scala.concurrent.ExecutionContext
```

Comment:
This seems unnecessary.

Reply:
I believe it's used by `createDummyContract` at the end of the class (since the suite no longer has its own `ExecutionContext`).

Reply:
If that wasn't the case, our tooling would detect it. :slightly_smiling_face: